vendor.conf

```diff
@@ -122,8 +122,8 @@ github.com/cyphar/filepath-securejoin v0.2.1
 gopkg.in/square/go-jose.v2 v2.1.3
 golang.org/x/time f51c12702a4d776e4c1fa9b0fabab841babae631
 github.com/cri-o/ocicni 84aa158d2bacf95147b2b0a89615dd665630f440
-github.com/projectatomic/libpod master
 github.com/containerd/continuity d8fb8589b0e8e85b8c8bbaa8840226d0dfeb7371
 github.com/modern-go/reflect2 05fbef0ca5da472bbf96c9322b84a53edc03c9fd
 github.com/modern-go/concurrent bacd9c7ef1dd9b15be4a9909b8ac7a4e313eec94
 github.com/ulikunitz/xz v0.5.4
+github.com/containers/libpod b4420e22fc838fd2bd9712d476656ed6e891d4c8
```

Comment:
looks like some kind of funkiness at the end of this line.

Reply:
That just means their is no new line at the end of vendor.conf.  Should not be a big deal.